api.md

```diff
@@ -1822,6 +1823,37 @@ ORDER BY day;
 
 ## Utilities/Statistics [](utilities)
 
+### timescaledb_information.data_node [](#timescaledb_information-datanode)
+
+Get information on data nodes. This function is specific to running
+TimescaleDB in a multi-node setup.
+
+|Name|Description|
+|---|---|
+| `node_name` | Data node name. |
+| `owner` | OID associated with the user who first created the data node. |
+| `options` | Options used when creating the data node. |
+| `node_up` | (BOOLEAN) Data node responds to ping. |
+| `num_dist_tables` | Number of distributed hypertables that use this data node. This metric
+is only available if a node is up. |
```

Comment:
You cannot include a line break here
![Screen Shot 2019-07-12 at 11 53 48 AM](https://user-images.githubusercontent.com/12288273/61141508-bbd9a580-a49b-11e9-9750-d20c5338d1f9.png)

Reply:
If you want multiline text in the table cell, use a `<br/>` element between the lines

Reply:
Removed all linebreaks.